ios/Podfile

```diff
@@ -12,8 +12,8 @@ target 'ProjectAccessibility' do
     :hermes_enabled => false
   )
 
-
-  pod 'react-native-config', :path => '../node_modules/react-native-config'
```

Comment:
Waarom is dit weggehaald?

Reply:
Goeie, zal zo wel ff op school kijken of de IOS dan wel nog werkt